build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/dsl/Publish.kt

```diff
@@ -77,11 +77,13 @@ fun Project.configurePublishing(repoName: String) {
         publications.all {
             if (this !is MavenPublication) return@all
 
+            val githubOrganization = if (repoName.equals("smithy-kotlin", ignoreCase = true)) "smithy-lang" else "awslabs"
+
```

Comment:
**Correctness**: This seems like the wrong location for the logic. Can't we just parameterize `githubOrganization` the same way we already do `repoName`?

Reply:
Sure